__tests__/interceptors/checkReRun.test.js

```diff
@@ -3,34 +3,58 @@ const CheckReRun = require('../../lib/interceptors/checkReRun')
 const Helper = require('../../__fixtures__/helper')
 require('object-dot').extend()
 
-test('#possibleInjection', () => {
-  let checkReRun = new CheckReRun()
-
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 1})
-  ).toBe(false)
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 2})
-  ).toBe(true)
-})
-
-test('#process', async () => {
+test('return if pre conditions are not met', async () => {
   let checkReRun = new CheckReRun()
   let context = Helper.mockContext()
 
   context.event = 'check_run'
-  context.payload.action = 'rerequested'
+  context.payload.action = 'opened'
```

Comment:
It might be clearer if we change the action to something that is valid for the event used in the test, i.e. `created` or `completed`

Reply:
done